src/features/Students/StudentsMetrics/index.jsx

```diff
@@ -2,28 +2,34 @@ import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { Card, CardGrid, ToggleButton } from '@edx/paragon';
 import { ToggleButtonGroup } from 'react-paragon-topaz';
-import { fetchMetricsData } from 'features/Students/data/thunks';
+import { fetchClassesMetricsData, fetchStudentsMetricsData } from 'features/Students/data/thunks';
 
 import 'features/Students/StudentsMetrics/index.scss';
 
 const StudentsMetrics = () => {
   const dispatch = useDispatch();
-  const stateMetrics = useSelector((state) => state.students.metrics.data);
+  const selectedInstitution = useSelector((state) => state.main.selectedInstitution);
```

Comment:
Maybe just `institution` instead of `selectedInstitution` is more clear
```suggestion
  const institution = useSelector((state) => state.main.selectedInstitution);
```